src/lib/Server/Output/ValueObjectVisitor/RestFieldDefinition.php

```diff
@@ -53,16 +53,21 @@ public function visit(Visitor $visitor, Generator $generator, $data)
             $visitor->setHeader('Accept-Patch', $generator->getMediaType('FieldDefinitionUpdate'));
         }
 
-        $generator->startAttribute(
-            'href',
-            $this->router->generate(
-                "ezpublish_rest_loadContentType{$urlTypeSuffix}FieldDefinition",
-                [
-                    'contentTypeId' => $contentType->id,
-                    'fieldDefinitionId' => $fieldDefinition->id,
-                ]
-            )
-        );
+        if ($data->path === null) {
+            $generator->startAttribute(
+                'href',
```

Comment:
`href` could be moved to a constant.

Reply:
I changed it to only generate value for `href` instead of repeating the same thing over and over, so now I think We don't need to give `href` to constant